cardano-api/api/model/skyline/response/balance_response.go

```diff
@@ -0,0 +1,15 @@
+package response
+
+import "math/big"
+
+type BalanceResponse struct {
+	Balance string `json:"balance"`
```

Comment:
BalanceResponse should not be Balance string, but Balance map[string]string, since it needs to return balances for multiple tokens